### ISSUE TITLE
[ENH] Prophet wrapper can't add multiple custom seasonalities

### ISSUE THREAD

**Issue by IlyasMoutawwakil** [alan-turing-institute/sktime#1374]
> **Is your feature request related to a problem? Please describe.**

The problem is in the sktime wrapped version of prophet. In Native Prophet you can add multiple seasonalities but in the sktime version you can only add one (other than the default seasonalities). 
I tried going around this without changing source code by calling model._forecaster (the prophet model instance) but it doesn't work since the instanciation is created during the fit and after the fit you can't do that as well.

> **Describe the solution you'd like**

A solution would be (what I did) to change the add_seasonality parameter from being a dictionary to a list of dictionaries (or maybe allow both and check for parameter type). The list should contain each seasonality's parameters to be added to the model.

I changed the lines from 47 to 50 in the file `sktime\forecasting\base\adapters\_fbprophet.py` :

```
# Add seasonality
if self.add_seasonality:
    for seasonality in self.add_seasonality:
        self._forecaster.add_seasonality(**seasonality)
```

> **Additional context**

I guess this is important because one of the things everyone loves about prophet is its agility and how you can use multiple seasonalities with different scale and Fourrier orders (the default seasonalities are fine but not good enough).
I actually was using native prophet with seasonalities that I added. To use your ensemble forecaster, I have to use the sktime version so that's when I had a problem.

**Comment by aiwalter** [comment]
Good point, would appreciate a PR.

**Comment by aiwalter** [comment]
closed by #1378
